src/pages/choose-account.tsx

```diff
@@ -1,49 +0,0 @@
-import React, { memo, useCallback, useEffect } from 'react';
```

Comment:
seems misconfigured, this is 100% used unless something changed

Reply:
https://github.com/blockstack/stacks-wallet-web/blob/main/src/routes.tsx#L7

Reply:
maybe I hadn't committed these changes yet? typecheck passes locally